docs/writing-stories/introduction.md

```diff
@@ -135,9 +135,9 @@ Refine this pattern by defining a master template for a component’s stories th
 
 <!-- prettier-ignore-end -->
 
-The template is reused across stories. Template.bind({}) makes a copy of the function which reduces code duplication. Similarly,`...Primary.args` makes a copy of the data, reducing data duplication.
+By introducing args into your component's stories, you're not only reducing the amount of code you need to write, but you're also decreasing data duplication, as shown by spreading the `Primary` story's args into the other stories.
 
-What’s more, you can import args to reuse when writing stories for other components. This is useful when you’re building composite components. For example, if we make a `ButtonGroup` story, we might remix two stories from its child component `Button`.
+What’s more, you can import `args` to reuse when writing stories for other components, and it's helpful when you’re building composite components. For example, if we make a `ButtonGroup` story, we might remix two stories from its child component `Button`.
 
 <!-- prettier-ignore-start -->
```

Comment:
_The `react/button-group-story.js.mdx` snippet wasn't modified in this PR, so I'm commenting here..._

This snippet is odd to me. `Pair.args.buttons = [{ ...ButtonStories.Primary.args }, { ...ButtonStories.Secondary.args }]` implies a few things:

1. Because there's no explicit render function, it must be using the implicit `(args) => <ButtonGroup {...args} />`
    - Which means `args.buttons` implies that ButtonGroup accepts a `buttons` prop when `children` would be much more idiomatic
1. Because the value of `args.buttons` is an array of objects (which are the props applied to each Button in the group), ButtonGroup must be iterating over that array and rendering the Button(s) itself
    - This is strange. ButtonGroup could simply accept Button(s) as `children`, instead.
    
I realize that this construct was likely chosen to illustrate args composition over anything else (and that it existed prior to CSF3), but now that the Template has been replaced by an implicit render function, I think this snippet might be confusing to viewers, who are likely assuming a more idiomatic ButtonGroup implementation and won't know how this concepts could apply to that.